From d4571eb653823d0e8936b35502fc38b7af9f6f04 Mon Sep 17 00:00:00 2001
From: sjorge <sjorge@blackdot.be>
Date: Tue, 14 Aug 2018 11:13:03 +0200
Subject: [PATCH] OS-7116 bhyve brand should accept bios and uefi values for
 bootrom (fix cstyle) Reviewed by: John Levon <john.levon@joyent.com> Reviewed
 by: Mike Gerdts <mike.gerdts@joyent.com> Approved by: Mike Gerdts
 <mike.gerdts@joyent.com>

---
 usr/src/lib/brand/bhyve/zone/boot.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usr/src/lib/brand/bhyve/zone/boot.c b/usr/src/lib/brand/bhyve/zone/boot.c
index 4a8b6562a2..ea0f8c15a6 100644
--- a/usr/src/lib/brand/bhyve/zone/boot.c
+++ b/usr/src/lib/brand/bhyve/zone/boot.c
@@ -424,7 +424,7 @@ add_lpc(int *argc, char **argv)
 				val = DEFAULT_BOOTROM_CSM;
 			} else if (strcmp(val, "uefi") == 0) {
 				val = DEFAULT_BOOTROM;
-                        }
+			}
 		}
 		if (snprintf(conf, sizeof (conf), "%s,%s", lpcdevs[i], val) >=
 		    sizeof (conf)) {
-- 
2.21.0

